doc/agama-security.md

```diff
@@ -0,0 +1,34 @@
+Agama Concepts
```

Comment:
Let's try to be consistent and use the `#`-based markdown notation. Not a big issue, but most of our documents are written in that way.

Reply:
ok, i planned to do markdown fine as last step